update-imported-docs/update-imported-docs.py

```diff
@@ -0,0 +1,253 @@
+        directory name or a file name.
+    :param repo_path:
+    :param repo_dir:
+    :param root_dir:
+    :param gen_absolute_links:
+    """
+    pattern = os.path.join(repo_dir, repo_path, src)
+    dst_path = os.path.join(root_dir, dst)
+
+    for src in glob.glob(pattern):
+        # we don't dive into subdirectories
+        if not os.path.isfile(src):
+            print("[Error] skipping non-regular path {}".format(src))
+            continue
+
+        content = ""
+        try:
+            with open(src, "r") as srcFile:
+                content = srcFile.read()
+        except Exception as ex:
+            print("[Error] failed in reading source file: ".format(ex))
+            continue
+
+        dst = dst_path
+        if dst_path.endswith("/"):
+            base_name = os.path.basename(src)
+            dst = os.path.join(dst, base_name)
+
+        try:
+            print("Writing doc: " + dst)
+            with open(dst, "w") as dstFile:
+                if gen_absolute_links:
+                    src_dir = os.path.dirname(src)
+                    remote_prefix = repo_path + "/tree/master"
+                    content = process_links(content, remote_prefix, src_dir)
+                if dst.endswith("kubectl.md"):
+                    print("Processing kubectl links")
+                    content = process_kubectl_links(content)
+                dstFile.write(content)
+        except Exception as ex:
+            print("[Error] failed in writing target file {}: {}".format(dst, ex))
+            continue
+
+
+def parse_input_args():
+    """
+    Parse command line argument
+    There is only one argument, 'config_file, which should be one of the YAML
+    files in this same directory
+    :return: parsed argument
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument('config_file', type=str,
+                        help="reference.yml to generate reference docs; "
+                             "release.yml to generate release notes")
+    return parser.parse_args()
+
+
+def main():
+    """The main entry of the program."""
+    if len(error_msgs) > 0:
+        for msg in error_msgs:
+            print(msg + "/n")
+        # sys.exit(-1)
+        return -2
+
+    # first parse input argument
+    in_args = parse_input_args()
+    config_file = in_args.config_file
+    print("config_file is {}".format(config_file))
+
+    curr_dir = os.path.dirname(__file__)
+    print("curr_dir {}".format(curr_dir))
+    root_dir = os.path.realpath(os.path.join(curr_dir, '..'))
+    print("root_dir {}".format(root_dir))
+
+    try:
+        # yaml.load has been deprecated
+        # https://github.com/yaml/pyyaml/wiki/PyYAML-yaml.load(input)-Deprecation
+        config_data = yaml.full_load(open(config_file, 'r'))
+    except Exception as ex:
+        print("[Error] failed in loading config file - {}".format(str(ex)))
+        return -2
+
+    os.chdir(root_dir)
+
```

Comment:
Line 190: nit: Is it required to change to root_dir?

Reply:
That's the original code, so I'm not inclined to change it at this time.